### PR TITLE
Add TuYa Clusters for TS0505A

### PR DESCRIPTION
**Pull request by MSchutt**
https://github.com/Koenkk/zigbee2mqtt/issues/4240

### CODE REVIEW

**Comment by MSchutt** [comment]
Please see https://github.com/Koenkk/zigbee-herdsman-converters/pull/1661

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -2087,6 +2089,15 @@ const Cluster: {
                     {name: 'transtime', type: DataType.uint8},
                 ],
             },
+            moveToHueAndSaturationBrightness: {
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts:2092]: I've checked the ZCL: https://github.com/Koenkk/zigbee-herdsman/blob/master/docs/Zigbee%20Cluster%20Library%20Specification%20v7.pdf and command 6 is the `moveToHueAndSaturation` and does not provide a brigthness:

image

Comment (MSchutt) [reply src/zcl/definition/cluster.ts:2092]: Yes I know, but unfortunately my TuYa LEDs don't seem follow the standard for whatever reason. Please see attached an image of a packet captured via Wireshark.
The payload contains FF which is the brightness in the end.

wireshark


Hue: 0x55 = 85
Saturation: 0xFE = 254
Transition time: 0x0000 = 0
Brightness: 0xFF = 255

Comment (Koenkk) [reply src/zcl/definition/cluster.ts:2092]: In that case change `moveToHueAndSaturationBrightness` to `tuyaMoveToHueAndSaturationBrightness`

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/1661

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**